### PR TITLE
#381 Remove warning log

### PR DESCRIPTION
**Pull request by uilianries**
Signed-off-by: Uilian Ries <uilian@khomp.com>

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -79,24 +79,20 @@ def package(self):
...
                 command = "install_name_tool -change %s @executable_path/../lib/%s %s" % (lib, lib, protoc)
-                self.output.warn(command)
                 self.run(command)
             libprotoc = os.path.join(self.package_folder, "lib", libprotoc)
             command = "install_name_tool -change %s @loader_path/%s %s" % (libprotobuf, libprotobuf, libprotoc)
-            self.output.warn(command)
             self.run(command)
             with tools.chdir(os.path.join(self.package_folder, 'lib')):
                 for l in glob.glob("*.dylib"):
                     command = 'otool -L %s' % l
```
Comment (SSE4) [inline conanfile.py]: you may remove otool invocation here and above

File: conanfile.py [hunk 2]
```diff
@@ -79,24 +79,20 @@ def package(self):
...
-                    self.output.warn(command)
```
Comment (SSE4) [inline conanfile.py]: you may remove otool invocation here

File: conanfile.py (hunk 1 above)
Comment (uilianries) [reply conanfile.py]: done!

File: conanfile.py (hunk 2 above)
Comment (uilianries) [reply conanfile.py]: done!

**Pull request merged**